### DISCUSSION TITLE
Current Solar Generation?

### DISCUSSION THREAD

**Discussion by tonywheeler60** [binsentsu/home-assistant-solaredge-modbus#135] [Q&A]
Firstly thanks - I've got my Modbus feeding data into HQ and it appears all to be working.  However, I cannot determine what entity to use to represent the current solar power generation value.

Any help would be much appreciated, as indeed would any pointers as to the meaning of all the returned Modbus entities.
BTW - I have a single battery, single inverter, single meter system.
TIA

Tony

**Comment by bramvandenbussche** [comment]
I too am struggling to make sense of all the meter values and trying to find how these correlate with what I'm getting from the app/API. +1

**Comment by mpredfearn** [comment]
https://github.com/binsentsu/home-assistant-solaredge-modbus/wiki/Using-Templated-Sensors-to-Calculate-Power-Flow-and-Energy
